content/blog/kissim-chemrxiv.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Explore kinase pocket similarities with KiSSim!"
+date: 2021-12-15T12:00:00+02:00
+author: dominique.sydow
+---
+
+Kinases are an important drug target since their dysregulation can cause servere diseases such as cancer, inflammation, and neurodegeneration. Finding selective drugs, however, is challenging due to the highly conserved binding sites across the roughly 500 human kinases, leading to unwanted side-effects. The underlying off-targets are often not trivial to predict and explain from a sequence-based perspective.
```

Comment:
```suggestion
Kinases are an important class of drug targets since their dysregulation can cause severe diseases such as cancer, inflammation, and neurodegeneration. Finding selective drugs, however, is challenging due to the highly conserved binding sites across the roughly 500 human kinases, which can lead to unwanted side-effects. The underlying off-targets are often not trivial to predict or to explain from a sequence-based perspective.
```

Reply:
maybe add sth like '... as given by the well known Manning tree [some link]' at the end